pkg/types/machinepools.go

```diff
@@ -50,6 +50,6 @@ type EC2RootVolume struct {
 // LibvirtMachinePoolPlatform stores the configuration for a machine pool
 // installed on libvirt.
 type LibvirtMachinePoolPlatform struct {
-	// QCOWImagePath
```

Comment:
Also, adding comments for your new properties will make it easier for folks to understand the config structure by reading [the godocs][1].

[1]: https://godoc.org/github.com/openshift/installer/pkg/types#LibvirtMachinePoolPlatform